content/doc/book/pipeline/syntax.adoc

```diff
@@ -807,6 +807,15 @@ provide when triggering the Pipeline. The values for these user-specified
 parameters are made available to Pipeline steps via the `params` object,
 see the <<parameters-example>> for its specific usage.
 
+Each parameter has a _Name_ and some sort of _Value_, depending on the parameter type.
+These name-value pairs will be exported as environment variables when the build starts, allowing subsequent parts of the build configuration (such as build steps) to access those values, e.g. by using the `+{PARAMETER_NAME}+` syntax (or `%PARAMETER_NAME%` on Windows).
+
+[NOTE]
+====
+This also implies that each parameter defined here should have a unique Name. For more information visit 
+link:https://github.com/jenkinsci/jenkins/blob/8a29e8858b49da45c0071f690d4ceebe8db2f18f/core/src/main/resources/hudson/model/ParametersDefinitionProperty/help.html[ParametersDefinitionProperty]
+====
```

Comment:
I think that the detail offered in this note is too broad for the section of documentation where it is placed and the linked page is not a usable location for most readers.  Most readers do not expect to read HTML markup in order to understand a concept.

I recommend it be removed from the pull request.
```suggestion

```

Reply:
Thanks @MarkEWaite for the review. The HTML markup contains information i think that is relevant to this PR maybe i can copy paste it somewhere where it can be seen by readers.
and the note can be shorten without referencing the link.
What are your thoughts on that?

Reply:
Thanks @MarkEWaite for the review. I can shorten the note and remove the link.